packages/block-library/src/site-title/index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Server-side rendering of the `core/site-title` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/site-title` block on the server.
+ *
+ * @return string The render.
+ */
+function render_block_core_site_title() {
+	return sprintf( '<h1>%s</h1>', get_bloginfo( 'name' ) );
```

Comment:
What about linking this to the home?

E.g.
```php
<h1>
	<a href="<?php echo esc_url( home_url( '/' ) ); ?>"><?php bloginfo( 'name' ); ?></a>
</h1>
```

Reply:
Maybe that should be an option/attribute?

Reply:
That's a very cool idea!
And maybe the `home_url` could be the default.

Reply:
Hi,

What happened with the site title linking to the home page? I can be done like this:

`function gutenberg_render_block_core_site_title() {
	return sprintf( '<h1 class="site-title"><a href="%s">%s</a></h1>', home_url( '/' ), get_bloginfo( 'name' ) );
}`

Cheers!

Reply:
That could be a nice toggleable feature. Feel free to open an issue or PR for it :smile: